_posts/2022-12-06-integrating-auth0-with-react.md

```diff
@@ -0,0 +1,307 @@
+---
+layout: post
+font: serif
+title: Integrating Auth0 Authentication into React
+date: 2022-12-06
+highlight: monokai
+author: Chris Howard
+image: img/blog/integrating-auth0-with-react/banner.jpg
+tags: Auth0 react authentication typescript
+description: 
+---
+
+Authentication and Authorization is objectively the most important part of any application, So why is it so often left to the end of development cycles or implemented in a slap dash manner?
```

Comment:
> Authentication and Authorization

You only cover Authentication in this post.  

Do you plan to do a follow up for authorization (which would be great)?

They are long enough to split into separate posts, but you should call out that you are only covering one half somewhere.  Either in the introduction or outro.